packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -502,6 +502,44 @@ class MethodChannelCamera extends CameraPlatform {
     }
   }
 
+  @override
```

Comment:
The changes to this file should be reverted; if there are sill third-party implementations using this (hopefully not), they should get the default behavior of nothing being supported, rather than throwing unsupported platform channel method exceptions.